test/spmd/test_xla_sharding.py

```diff
@@ -481,6 +481,41 @@ def test_xla_sharded_hlo_dump(self):
     # scalar 5 should be replicated
     self.assertIn('%p0.2 = f32[] parameter(0), sharding={replicated}', hlo)
 
+  @unittest.skipIf(xr.device_type() == 'TPU', "Crash on TPU v2")
```

Comment:
Confirmed working on V4-8, will remove this once I incorporate V2 support in HybridMesh